ddl/db_test.go

```diff
@@ -85,7 +85,6 @@ func setUpSuite(s *testDBSuite, c *C) {
 	var err error
 
 	s.lease = 100 * time.Millisecond
-	session.SetSchemaLease(s.lease)
```

Comment:
Many tests rely on the lease value, it is best to check the use of this value in the test.

Reply:
This change affects db_test.go
I find the failed test is db_test.go, so I change it.
If there are other tests found to fail on SchemaLease, we can also update them.